compiler/src/dotty/tools/dotc/transform/LinkScala2Impls.scala

```diff
@@ -88,8 +94,13 @@ class LinkScala2Impls extends MiniPhase with IdentityDenotTransformer { thisPhas
     val implName = ImplMethName(meth.name.asTermName)
     val cls = meth.owner
     if (cls.isAllOf(Scala2xTrait))
-      if (meth.isConstructor)
+      if (meth.isConstructor) {
+        if (ctx.settings.scalajs.value) {
+        // Is this the static call?
         cls.info.decl(nme.TRAIT_CONSTRUCTOR).symbol
```

Comment:
I think this is incorrect but haven't been able to figure out the correct call, any suggestions?